packages/ui-a11y-utils/src/KeyboardFocusRegion.ts

```diff
@@ -160,6 +167,45 @@ class KeyboardFocusRegion {
     }
   }
 
+  tryFocusOnPositionedElement(
+    positionContainer: Element,
+    focusElement: HTMLElement,
+    tries = 3
```

Comment:
Arbitrary number. I tried this on a very beefy page and it never run out of tries.